packages/api/bin/serve.js

```diff
@@ -145,11 +141,11 @@ async function createDBRecords(stackName, user, knexOverride) {
   const rulePgModel = new RulePgModel();
 
   await initializeLocalElasticsearch(stackName);
+  await serveUtils.resetPostgresDb();
```

Comment:
`serveUtils` is a good idea. 

What was happening here without these changes? Were we completely failing to populate PG records? I see a dynamo reference or two but it looks like modifications were necessary on the PG side as well. Wondering if these changes are intended to make the fake records more useful or if they're actually making something broken work.

(Again, I apologize for not being able to test this. Serving on an M1 machine hits a connection hangup on localstack before any of this)

Reply:
Both - the changes were needed as precursor records weren't being populated correctly, there were additional dynamo references.       It clearly hasn't been working for some time given the nature of the failures.

Reply:
Interesting. It seems like this would date back to very early in RDS development. Maybe we weren't populating PG records correctly and it wasn't apparent until Phase 2 when the reads switched over.